tests/std/tests/P0896R4_views_reverse/test.cpp

```diff
@@ -230,6 +230,63 @@ constexpr bool test_one(Rng&& rng, Expected&& expected) {
         }
     }
 
+#if _HAS_CXX23
+    // Validate view_interface::cbegin
+    static_assert(CanMemberCBegin<R>);
+    {
+        const same_as<const_iterator<reverse_iterator<iterator_t<V>>>> auto ci = r.cbegin();
+        if (!is_empty) {
+            assert(*ci == *begin(expected));
+        }
+
+        if constexpr (copyable<V>) {
+            auto r2                                                                 = r;
+            const same_as<const_iterator<reverse_iterator<iterator_t<V>>>> auto ci2 = r2.cbegin();
+            assert(r2.cbegin() == ci2);
+            assert(r2.cbegin() == ci2);
+            if (!is_empty) {
+                assert(*ci2 == *ci);
+            }
+        }
+
+        static_assert(CanMemberCBegin<const R> == common_range<Rng>);
+        if constexpr (common_range<Rng>) {

```

Comment:
Style: I would generally expect the `if constexpr` to check `CanMemberCBegin<const R>` here, since that's the thing we're interested in doing, and that's the pattern in other tests.

Occurs below on line 284.